linux_os/guide/system/accounts/accounts-restrictions/password_storage/no_empty_passwords/oval/shared.xml

```diff
@@ -16,7 +16,7 @@
   </ind:textfilecontent54_test>
   <ind:textfilecontent54_object id="object_no_empty_passwords" version="1">
     <ind:filepath>/etc/pam.d/system-auth</ind:filepath>
-    <ind:pattern operation="pattern match">\s*nullok\s*</ind:pattern>
+    <ind:pattern operation="pattern match">^[^#]*nullok.*$</ind:pattern>
```

Comment:
This will match also if nullok is a prefix of something, eg. nullokabc.

Reply:
@jan-cerny updated. Added word boundaries to be sure that `nullok` is standalone word